test/integration/fs-options-non-evaluable/index.js

```diff
@@ -0,0 +1,5 @@
+const dir = __dirname
```

Comment:
This is weird. This does not work :

```js
module.exports = require('fs').readFileSync(__dirname + '/test.txt', {
  encoding: (typeof Date.now()).replace(/number/, 'utf-8')
})

```

While `const dir = __dirname` works. I think it may be (not sure at all) a [babylon-walk](https://github.com/pugjs/babylon-walk) bug, but I'm not really sure about that. I checked the `Identifier` visitor and it is not called with the node, but it seems to be in the AST. I can't reproduce it outside of this test 😕